week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,19 +20,36 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    while (this.volume < 100) {
+
+      this.volume++;
+    }
+    console.log(this.volume);
   },
+  // calling this function should pour your bottle full (volume = 100);
   pour: function () {
+    this.volume1 = this.volume + 10
+    console.log(this.volume1);
+

```

Comment:
do you need to put a condition to ensure it's not higher that 100? (can you pour over 100?)

Reply:
is there an easier way to write this
this.volume=this.volume+10?